Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: help lint scan clean
+
+# Default target when just running 'make'
+.DEFAULT_GOAL := help
+
+# Python interpreter to use
+PYTHON := python3
+
+.PHONY: lint
+lint:  ## Run code linting (flake8, black, isort)
+	ruff check --fix
+
+.PHONY: format
+format:  ## Format code using black and isort
```

Comment:
These comments seem copied over - possibly we could update them to match our actual build?